COCO_DATA_FORMAT.md

```diff
@@ -236,7 +236,7 @@ It is a generic variation of an image-text dataset, where the input consists of
   - images, image is optionally associated with a metadata dictionary which stores the text attributes of interest for the image. For example, image is a product catalog image: `{'metadata': {'catalog': true}}`, capture location of an image: `{'metadata': {'location': 'street'}}`, information of the assembly component captured in image of a defect detection dataset: `{'metadata': {'name': 'Hex Head Lag Screw', 'type': '3/8-inch x 4-inch'}}`  
   - text (optional), a dictionary with keys being field names e.g. `{'text': {'question': 'a specific question related to the images input'}}`
 - output:
-  - key-value pairs, a dictionary with keys being the attributes of interest
+  - key-value pairs, a dictionary with keys being the attributes of interest, values being dictionary that stores the actual attribute value in "value" and optionally a list of grounded bboxes in "groundings". "groundings" are for single-image annotation only. Each bbox follows [BBox Format](#bbox-format).
 
 The dataset format is a simple variation of COCO, where `image_id` of an annotation entry is replaced with `image_ids` to support multi-image annotation. In each annotation entry, `key_value_pairs` field is required, `text` is optional. In each image entry, `metadata` is optional. Below is an example of multi-image question answering.
```

Comment:
Should rename `key_value_pairs` to `fields` here as well if we change it in the annotations.